specificatie/domain.yaml

```diff
@@ -203,6 +203,37 @@ components:
                         De identificaties van de stukdelen (paragrafen in een akte met een rechtsfeit) waarin dit beslag is vermeld
           items:
             type: "string"
+    DeelEnNummer:
+      type: "object"
+      description: |
+                    Deel en nummer identificeren een Ter Inschrijving Aangeboden stuk door gebruik te maken van vier gegevenselementen.
+                    Waardelijst in deze component :
+                      [reeks(http://www.kadaster.nl/schemas/waardelijsten/Reekscode/)
+                      [registercode](http://www.kadaster.nl/schemas/waardelijsten/Registercode/)
+                      [soortRegister](http://www.kadaster.nl/schemas/waardelijsten/SoortRegister/)
+      properties:
+        deel:
+          description: |
+                        Identificatie van het stuk binnen zijn soort.
+          type: "string"
+        nummer:
+          description: |
+                        Volgnummer van het stuk.
+          type: "string"
+        reeks:
```

Comment:
Is dit een attribuut dat moet worden ingevuld in KIKInzage voor het ophalen van een akte (BLOB)?

Reply:
@kad-pothot Ik denk dat jij degene bent die bovenstaande vraag van Cathy kan (laten) beantwoorden.

Reply:
@CathyDingemanse  Ja, die heb je nodig, net als reeks (voor oudere stukken)